index.html

```diff
@@ -5,10 +5,16 @@
     <title>Consumer Side</title>
     <link rel="stylesheet" href="css/reset.css">
     <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Loved+by+the+King" rel="stylesheet">
   </head>
 
+  
   <body>
-    <h1>Test Consumer Headline</h1>
+    <div id="splashImage">
+      <img id="salmon" src="img/salmon.png"></div>>
+      <div id="logoText">Pat's Salmon Cookies</div>
+    </div> 
+    
```

Comment:
You're missing the required basic business info on the homepage. From the assignment:
"Include all of the typical stuff that you'll find on the home page of a business: locations, hours, contact information, some text about how awesome the business is, etc. Be creative, and again, think about what is meaningful to a typical end user."